### PR TITLE
add lmsys_chat_1m dataset

### PR DESCRIPTION
**Pull request by manandey**
c.c. @shayne-longpre. FYI. This is a gated dataset, so can't be downloaded without getting access. All tests are passing.

### CODE REVIEW

File: constants/license_classes.json [hunk 1]
```diff
@@ -225,5 +225,10 @@
         "All",
         "1",
         "0"
+    ],
+    "LMSYS-Chat-1M Dataset License":[
```
Comment (shayne-longpre) [inline constants/license_classes.json:229]: @rmahari

File: data_summaries/lmsys_chat_1m.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2309.11998",
+        "Semantic Scholar Corpus ID": 262084217,
+        "Collection": "lmsys_chat_1m",
+        "Collection URL": "https://huggingface.co/datasets/lmsys/lmsys-chat-1m",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Question Answering"
```
Comment (shayne-longpre) [inline data_summaries/lmsys_chat_1m.json:18]: This dataset is super diverse, including many creative tasks. The "Task Categories" attribute is kind of an estimate, but try to add 6-12 more entries from a variety of categories in the constants file, that seems to represent the data. I'd imagine most categories are well covered by this dataset.

File: data_summaries/lmsys_chat_1m.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        "Dataset URL": "https://huggingface.co/datasets/lmsys/lmsys-chat-1m",
+        "GitHub URL": "",
+        "Hugging Face URL": "https://huggingface.co/datasets/lmsys/lmsys-chat-1m",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2309.11998",
+        "Semantic Scholar Corpus ID": 262084217,
+        "Collection": "lmsys_chat_1m",
+        "Collection URL": "https://huggingface.co/datasets/lmsys/lmsys-chat-1m",
+        "Languages": [
+            "English"
```
Comment (shayne-longpre) [inline data_summaries/lmsys_chat_1m.json:15]: from paper: "the top five languages are English, Portuguese,
Russian, Chinese, and Spanish"

File: data_summaries/lmsys_chat_1m.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        ],
+        "Task Categories": [
+            "Question Answering"
+        ],
+        "Text Sources": [],
```
Comment (shayne-longpre) [inline data_summaries/lmsys_chat_1m.json:20]: "crowdsourced" as well

File: data_summaries/lmsys_chat_1m.json [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+        ],
+        "Human Annotation": "No",
+        "Derived from Datasets": [],
+        "Creators": ["UC Berkeley", "UC San Diego", "Carnegie Mellon University", "Stanford University", "MBZUAI"
+        ],
+        "Licenses": [
+            {
+                "License": "LMSYS-Chat-1M Dataset License",
+                "License URL": "https://huggingface.co/datasets/lmsys/lmsys-chat-1m"
+            }
```
Comment (shayne-longpre) [inline data_summaries/lmsys_chat_1m.json:39]: Also, OpenAI, Llama license (https://github.com/Data-Provenance-Initiative/Data-Provenance-Collection/blob/main/constants/license_classes.json#L224), and Anthropic license (may need to add).

**Review by shayne-longpre — APPROVED** [review]
A few minor comments, one which may require Robert, otherwise good!

File: constants/license_classes.json (hunk 1 above)
Comment (rmahari) [reply constants/license_classes.json:234]: I think it's All uses, no share alike, no attribution.

**Pull request merged**